### ISSUE TITLE
Using groupedByWeek() method returns Carbon\Exceptions\UnknownMethodException error after updating to 5.0.4

### ISSUE THREAD

**Issue by danystad** [solspace/craft3-calendar#292]
**Describe the bug or issue you're experiencing**

**Steps to reproduce**

1. Updated Calendar plugin from version 5.0.3 to 5.0.4
2. Page template that uses the groupedByWeek() method on the calendar.events template query now returns the following error:

```
Carbon\Exceptions\UnknownMethodException

Method Carbon\Carbon::setWeekStartsAt does not exist.
```

**Craft & Plugin Info (please complete the following information):**

- Craft Version: 4.9.5
- Calendar Version: 5.0.4
- Calendar Edition: pro
- Fresh Install or Upgrade: Updated Calendar plugin from version 5.0.3 to 5.0.4

**Comment by kjmartens** [comment]
This should now be resolved in Calendar 5.0.6+. 🙂

**Comment by danystad** [comment]
5.0.6 fixed the above, but I'm now seeing new errors with the example template code for [Grouping Events by Month/Week/Day](https://docs.solspace.com/craft/calendar/v5/templates/queries/events/#grouping-events-by-month-week-day) from the Calendar documentation.

I opened a new issue #303
